USER_MANUAL.md

```diff
@@ -336,6 +336,16 @@ These tests can only be run automatically. To run them:
   * Create a Cloud deployment in `unreal_gdk_starter_project` named `client_travel_gym`.
   * Launch the game in PIE mode, connecting to the corresponding Cloud Deployment.
   * Press K to trigger a ClientTravel for the PlayerController to the same deployment. It should be visible in PIE as your position in the map will be reset.
+  
+##### Cross-Server possession gym
+* Functional tests the Controller remote possess the Pawn
+* How to test:
+  * Set the current map to `PossessionGym`
+  * Set the `GameMode Override` to `CrossServerPossessionGameMode`(World Settings > Game Mode)
```

Comment:
What content is being reused between the possession tests?
I think we should create a separate map, so the user doesn't need to change anything.

Reply:
Good idea!

Reply:
After refactoring, those 2 NOTEs can be removed